.gitignore

```diff
@@ -18,6 +18,7 @@
 *.rej
 *.swp
 *~
+*.bak
```

Comment:
I have memories of this being discussed before (or maybe it was about `*.rej`): backup files should show up in `git status` so that the developer can have a look at the new and backup files to see if something is missing, and if satisfied delete the backup.  Hiding them defeats their usefulness.  You could argue that it should be ignored to ensure `git add .` doesn’t add it to the next commit, but developers should not add or commit files blindly, and we do have code review to catch these.

```suggestion
```